stable/node-problem-detector/README.md

```diff
@@ -31,6 +31,7 @@ $ helm delete my-release
 ## Configuration
 
 Custom System log monitor config files can be created, see [here](https://github.com/kubernetes/node-problem-detector/tree/master/config) for examples.
+Custom Plugin monitor config files have to be passed under `settings.custom_plugin_monitors`
```

Comment:
Can you remove this line and put it in the table of parameters below?

Reply:
Done.